app/web/template/layout/page.gohtml

```diff
@@ -157,6 +157,11 @@
                     {{ tr .App "toggleTransKeys" }}
                   </a>
                 </li>
+                <li class="govuk-footer__inline-list-item">
+                  <a class="govuk-footer__link" href="testing-certificate-provider-start">
+                    Certificate provider start
```

Comment:
this should be deleted when we get the email done, I don't like it

Reply:
I still have a strong desire for some kind of admin view somewhere that we can hide these things in (including the translations toggle). I've usually relied on user roles for this but as we scope the roles to types rather than prop on a user type we'd need a different approach for modernise...